web-ng/src/app/components/layer-dialog/layer-dialog.component.ts

```diff
@@ -24,11 +24,13 @@ import { Form } from '../../shared/models/form/form.model';
 import { Subscription } from 'rxjs';
 import { DataStoreService } from '../../services/data-store/data-store.service';
 import { Router } from '@angular/router';
-import { FormBuilder, FormGroup } from '@angular/forms';
-import { FieldType } from '../../shared/models/form/field.model';
+import { FormBuilder, FormGroup, FormArray } from '@angular/forms';
+import { FieldType, Field } from '../../shared/models/form/field.model';
 import { StringMap } from '../../shared/models/string-map.model';
 import { Map } from 'immutable';
 
+const layerColor = '#ff9131';
```

Comment:
This name implies this represents the layer color (as if there were one, fixed color), whereas it actually represents the default one used when there is none set, so DEFAULT_LAYER_COLOR would be a more appropriate name imo. (Caps+underscore since it's a top-level constant.